cinema/views.py

```diff
@@ -101,8 +105,20 @@ def get_serializer_class(self):
         if self.action == "retrieve":
             return MovieDetailSerializer
 
+        if self.action == "upload_image":
+            return MovieImageSerializer
+
         return MovieSerializer
 
+    @action(methods=["POST"], detail=True, url_path="upload_image")
```

Comment:
```suggestion
    @action(methods=["POST"], detail=True)
```
url_path is redundant because function named the same